### DISCUSSION TITLE
使用v1打包出的项目目录结构多了一层文件夹，这里可以通过配置取消吗？

### DISCUSSION THREAD

**Discussion by mokiYang** [vuepress-theme-hope/vuepress-theme-hope-v1#47] [Q&A] [answered]
例如，原本 md 文档的目录结构如下
├── docs
│   ├── guide
│   │   └── start.md

在构建后生成的 html 资源的目录结构如下
├── docs
│   ├── guide
│   │   ├── start
│   │   │   └── index.html

但由于在嵌入md链接时，嵌入的是xxx/guide/start.md，项目编译后对应的 html 文件中生成的 a 标签链接就会为 xxx/guide/start.html，这导致了使用右键新标签页打开链接时，会使得页面 not found，（但在当前页面打开并不会，或许是因为cleanUrl机制）

想问问是否有配置项可以控制不多余生成一层目录？因为在没有使用这个主题前，使用 vuepress 生成的目录也是正常的
预期生成的目录结构为：
├── docs
│   ├── guide
│   │   └── start.html

**Comment by Mister-Hope** [comment, marked as answer]
cleanUrl 是可以配置的 https://vuepress-theme-hope.github.io/v1/config/theme/plugin.html#cleanurl 

normalSuffix 设置为 `.html` 就是默认行为了

**Reply by Mister-Hope** [reply]
顺便一提的是 V1 已经停止维护，除非你的文档与 Vue2 强绑定，否则你应该使用 V2

**Reply by mokiYang** [reply]
好的非常感谢，当前v2还在发布beta版本，在它更新到稳定版后会进行更换
